internal/controllers/project/project_reconciler_function_test.go

```
@@ -877,6 +876,7 @@ var _ = Describe("Deletion Cleanup", func() {
 	})
 })
 
+//go:fix inline
 func strPtr(s string) *string {
```

golangci-lint (2.12.2)

[error] func strPtr is unused

(unused)